site/pages/create-consequence-operator/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -274,10 +302,29 @@ export const getServerSideProps = async (
 
     if (ctx.res) destroyCookieOnResponseObject(COOKIES_CONSEQUENCE_OPERATOR_ERRORS, ctx.res);
 
-    const operators = await fetchOperators({ adminAreaCodes: session.adminAreaCodes ?? ["undefined"] });
+    const operatorsData = await fetchOperators({ adminAreaCodes: session.adminAreaCodes ?? ["undefined"] });
+    const uniqueOperators: Operator[] = [];
+    const uniqueOperatorNames: Set<string> = new Set();
+
+    operatorsData.forEach((operator) => {
+        if (operator.mode === "bus" || operator.mode === "") {
```

Comment:
Could we use VehicleMode interface so VehicleMode.bus

Reply:
done